components/automate-gateway/handler/authz_test.go

```diff
@@ -44,26 +46,26 @@ func TestIntrospectAll(t *testing.T) {
 	}{
 		"one response pair, mapped": {
 			&authz.FilterAuthorizedPairsResp{Pairs: []*authz.Pair{
-				{Resource: "iam:policies", Action: "iam:policies:create"},
+				{Resource: "notifications:rules", Action: "notifications:notifyRules:create"},
```

Comment:
Because this file is testing v1 endpoints. You will likely remove or replace it as part of the introspection card you are working on. I removed the v1 policy endpoints so I had to replace them with some other v1 endpoint that exists still for now.